chipsec/cfg/8086/pch_c60x.xml

```diff
@@ -7,6 +7,15 @@ XML configuration file for C600 series PCH
   https://ark.intel.com/products/series/98463/Intel-C600-Series-Chipsets
 -->
 
+  <!-- #################################### -->
+  <!--                                      -->
+  <!-- Information                          -->
+  <!--                                      -->
+  <!-- #################################### -->
+  <info>
+    <sku did="0x1D41" name="C600" code="PCH_C60x" longname="Intel C600/X79 series PCH" />
+  </info>
```

Comment:
Are `code` and `platform` case sensitive?  The 'x' is lower case.

Reply:
they aren't but for consistency will make them match